config/layers.json

```diff
@@ -0,0 +1,90 @@
+{
+  "popupaltion_below_poverty": {
+    "title": "Population below national poverty line (%)",
+    "type": "wms",
+    "server": "geoserver",
```

Comment:
Is this enough config here? I'd imagine we'll probably need a server address as well as the type of server...

edit: nm, saw the `path` field below. But I think it should be more clear on first inspection what `server` vs. `path` means... (e.g. `server_type` & `uri`?)